doc/architecture/decisions/0019-define-and-run-cronjobs-via-kubernetes.md

```diff
@@ -0,0 +1,19 @@
+# 19. Define and run cronjobs via Kubernetes
+
+Date: 2023-01-19
+
+## Status
+
+Accepted
+
+## Context
+
+Prior to this decision, the DDS has defined the cronjobs in a module within the app, and they have been run via Flask-APScheduler. This worked without any issues for a while, but when the deployment setup on the cluster changed we started getting errors during for example the project status transitions which are run every day at midnight. After some investigating we noticed that the issues are due to there being two replicas with two worker processes each, giving a total of four cronjobs which run simultaneously. I order to avoid these issues now and in the future, we are now defining the current cronjobs as Flask commands and setting up the cronjobs in k8s.
```

Comment:
`Prior to this decision, the DDS has defined the cronjobs in a module within the app, and they have been run via Flask-APScheduler. This worked without any issues for a while, but when the deployment setup on the cluster changed we started getting errors during for example the project status transitions which are run every day at midnight. After some investigating we noticed that the issues are due to there being two replicas with two worker processes each, giving a total of four cronjobs which run simultaneously. I order to avoid these issues now and in the future, we are now defining the current cronjobs as Flask commands and setting up the cronjobs in k8s.`
can be smth like:
`Prior to this decision, the DDS has defined the cronjobs in a module within the app, and they have been run via Flask-APScheduler. This worked without any issues for a while, but when the production deployment setup on the cluster was scaled up to 2 replicas, we started getting errors during for example the project status transitions which are run every day at midnight. After some investigating we noticed that the issues are due to there being two replicas with two worker processes each, giving a total of four cronjobs which run simultaneously. I order to avoid these issues now and in the future, we are now defining the current cronjobs as Flask commands and setting up the cronjobs in k8s.
`